src/buildingriskassessment/navigators/SiteMaintenanceManagerNavigator.js

```diff
@@ -170,6 +171,10 @@ const SiteMaintenanceManagerNavigator = () => {
                 name={navigationRoutes.USERPROFILE}
                 component={UserProfileScreen}
             />
+            <SiteMaintenanceManagerDrawer.Screen
```

Comment:
If we use a `Drawer.Screen` here, we don't have the ability to add a `options` prop to the screen, which would let us see a screen title on the header bar when the screen loads. Let's use a `Stack` navigator here instead. Instead of passing the component directly into the Drawer, create a function which will return a `Stack.Navigator` with a `Stack.Screen` of the Maintenance Manager Calendar. Inside of the Drawer, you'll need to use the `children` prop instead of `component`. There's examples of this in other modules.